File: sentimental-mario-less/mario.py
from cs50 import get_int

# Asking for height between 1 and 8
height = get_int("Height: ")
while height < 1 or height > 8:
    print("Input a interger between 1 and 8")
    height = get_int("Height: ")

# Block building loop
for i in range(1, height + 1):
    print(" " * (height - i) + "#" * i)

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Asking for height between 1 and 8
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height<1 | height> 8);
    // Block building loop
    for (int row = 0; row < height; row++)
    {
        for (int col = 0; col < height; col++)
        {
            if (row + col < height - 1)
                printf(" ");
            else
                printf("#");
        }
        printf("\n");
    }
}
